README.md

```diff
@@ -46,16 +46,20 @@ This control uses [Google Material Icons](https://design.google.com/icons) by de
 Example usage:
 
 ```
-var control = L.control.zoomBox({
-    modal: true,  // If false (default), it deactivates after each use.  
-                  // If true, zoomBox control stays active until you click on the control to deactivate.
-    // position: "topleft",                  
-    // className: "customClass"  // Class to use to provide icon instead of Font Awesome
-    // title: "My custom title" // a custom title
-});
+var options = {
+    modal: true,
+    title: "Box area zoom"
+};
+var control = L.control.zoomBox(options);
 map.addControl(control);
 ```
-
+Options:
+- *modal*: (boolean, default = `false`); if false, it deactivates after each use; if true, zoomBox control stays active until you click on the control again to deactivate options (like a toggle button)
+- *position*: (string, default = `"topleft"`); position of the control
+- *title*: (string, default = `"Zoom to specific area"`) title attribute for the control
+- *content*: (string, default = `""`) html content to set as `innerHTML` of the control link
```

Comment:
@Azmisov either more direction to the user is required, or more handling of custom content is required.

When I use `content: '<div>ZOOMBOX</div>'`, I get:
![image](https://user-images.githubusercontent.com/3375604/32277103-9edc5c5c-bece-11e7-93e8-51a5cc89f246.png)

What should it look like instead?

Can you also please add an example to `examples/customContent.html` that demonstrates this functionality?  (just copy and slightly modify `index.html`).

Reply:
Yeah that was the intention, that you can add text or an svg/img tag inside. It still uses the regular zoombox container classname, and whatever classname you specified for the control link. So you need to style it explicitly. I changed the readme to make it clearer you'll need to style via css. Also added an example of using an SVG icon.